android/src/main/kotlin/com/pagecall/pagecall_flutter/FlutterPagecallView.kt

```diff
@@ -31,15 +32,29 @@ class FlutterPagecallView(
 
         WebView.setWebContentsDebuggingEnabled(debuggable)
 
-        pagecallWebView = PagecallWebView(context).apply {
-            listenMessage {
+        pagecallWebView = PagecallWebView(context)
+
+        loadPage()
+
+        pagecallWebView.setListener(object: PagecallWebView.Listener {
+            override fun onLoaded() {
                 Handler(context.mainLooper).post {
-                    channel.invokeMethod("onMessageReceived", it)
+                    channel.invokeMethod("onLoaded", null)
                 }
             }
-        }
 
-        loadPage()
+            override fun onMessage(message: String) {
+                Handler(context.mainLooper).post {
+                    channel.invokeMethod("onMessage", message)
+                }
+            }
+
+            override fun onTerminated(reason: TerminationReason) {
+                Handler(context.mainLooper).post {
+                    channel.invokeMethod("onTerminated", reason.toString())
```

Comment:
[Minor]
iOS와 똑같이 동작하게 만드는 것이 좋을 것 같습니다.
```suggestion
                    String reasonString = reason.getValue();
                    if (reason == TerminationReason.OTHER) {
                      reasonString = reason.getOtherReason();
                    }
                    channel.invokeMethod("onTerminated", reasonString)
```
이렇게 하면 iOS처럼 Reason.Other일 때 "other"이 아니라 "external"을 얻어낼 수 있습니다.

Reply:
[fix: pass terminate reason on android](https://github.com/pagecall/flutter-pagecall/pull/3/commits/0484017061ebb44552ad93e147327bfbd655a941) 에서 적용했고 잘 작동하는 것 확인했습니다.

Reply:
아아 굳굳